public/app/features/dashboard/dashgrid/DashboardGrid.tsx

```diff
@@ -153,8 +164,11 @@ export class DashboardGrid extends PureComponent<Props> {
 
     this.props.dashboard.sortPanelsByGridPos();
 
-    // Call render() after any changes.  This is called when the layout loads
-    this.forceUpdate();
+    // Call render() after any changes.  This is called when the initial layout loads
+    if (this.state.shouldForceUpdate) {
```

Comment:
Got a bit confused with the name of this state property. Was expecting it to be modified when force update is needed, but apparently it's only to indicate initial forceUpdate. Maybe we could call it `isInitialized` or sth like that?

Reply:
That's definitely a better name 🙌🏾